com.unity.visualeffectgraph/Editor/GraphView/Views/VFXPaste.cs

```diff
@@ -85,7 +106,7 @@ void PasteBlocks(VFXView view, ref SerializableGraph serializableGraph, List<VFX
             }
             else
             {
-                Debug.LogError(m_BlockPasteError.text);
+                Debug.LogWarning(m_BlockPasteError.text);
```

Comment:
This is not a error, rather an action that is not allowed. But with my changes it should not happen anymore